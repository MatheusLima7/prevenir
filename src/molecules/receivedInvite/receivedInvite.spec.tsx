import React from "react";
import { render } from "@testing-library/react-native";

import ReceivedInvite from "./receivedInvite";

describe("ReceivedInvite Component", () => {
  it("should render component correctly", () => {
    const container = render(
      <ReceivedInvite title={"title"} subtitle={"subtitle"} />
    );

    expect(container).toMatchSnapshot();
  });
});
