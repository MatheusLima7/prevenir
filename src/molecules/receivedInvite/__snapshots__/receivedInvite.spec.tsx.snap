// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReceivedInvite Component should render component correctly 1`] = `
<View
  style={
    {
      "marginBottom": 80,
      "marginTop": 110,
      "textAlign": "center",
    }
  }
>
  <Text
    center={true}
    size="small"
    style={
      [
        {
          "color": undefined,
          "display": "flex",
          "fontFamily": "Quicksand-Regular",
          "fontSize": 14,
          "fontWeight": "400",
          "lineHeight": 20,
          "textAlign": "center",
        },
        {},
      ]
    }
    testId="text-component"
    type="text"
  >
    title
  </Text>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <Text
      center={true}
      color="blue"
      size="medium"
      style={
        [
          {
            "color": "#0081A7",
            "display": "flex",
            "fontFamily": "Quicksand-SemiBold",
            "fontSize": 20,
            "fontWeight": "600",
            "lineHeight": 30,
            "textAlign": "center",
          },
          {},
        ]
      }
      testId="text-component"
      type="title"
    >
      subtitle
    </Text>
  </View>
</View>
`;
